flusher.go

```diff
@@ -40,20 +68,53 @@ func (s *Server) Flush(interval time.Duration, metricLimit int) {
 			finalMetrics = append(finalMetrics, g.Flush()...)
 		}
 		for _, h := range wm.histograms {
+			finalMetrics = append(finalMetrics, h.Flush(interval, percentiles)...)
```

Comment:
Still a bit confused here as to why histograms are being flushed instead of forwarded…